src/app/modules/project/add/manual/manual.component.html

```diff
@@ -66,6 +66,15 @@ <h3>Description*</h3>
           </div>
         </div>
       </div>
+      <div class="row">
+        <div class="col-12">
+          <div class="project-icon-picker">
+            <h3>Project icon</h3>
+            <app-file-uploader [acceptMultiple]="acceptMultiple" [acceptedTypes]="acceptedTypes">
```

Comment:
Why do we need the option to upload multiple file uploads for a project icon?

Reply:
In the backend discussion, Max & Ruben mentioned that we probably want to support upload .pdf, .doc files, and such later on. I figured I may as well implement it now.

Reply:
Indeed it is really nice to have functionality. Even though we are not using it at the moment, it ensures in the future that we can support multi file uploads in a different part of the app.